packages/arb-validator-core/gobridge/challenge.go

```diff
@@ -99,24 +95,39 @@ func (c *challenge) challengerResponded() {
 
 }
 
+func (c *challenge) resolveChallenge(winner common.Address, loser common.Address) {
+	rollup, ok := c.client.rollups[c.contractAddress]
+	if ok {
+		winningStaker := rollup.rollup.stakers[winner]
+		winningStaker.inChallenge = false
+		transferEth(c.client.goEthdata, winner, loser, rollup.rollup.chainParams.StakeRequirement)
```

Comment:
`StakeRequirement/2` should be transferred from the rollup chain to the winner, and no other funds should be transferred.

Reply:
I had this correct but somehow in cleanup before push it got deleted. Added back in and fixed.